client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/constraint/DbTableUniqueConstraint.java

```diff
@@ -0,0 +1,17 @@
+package org.evomaster.client.java.controller.internal.db.constraint;
+
+import java.util.List;
+
+public class DbTableUniqueConstraint extends DbTableConstraint {
+
+    private final List<String> uniqueColumnNames;
+
+    public DbTableUniqueConstraint(String tableName, List<String> uniqueColumnNames) {
+        super(tableName);
+        this.uniqueColumnNames = uniqueColumnNames;
+    }
```

Comment:
in general, unless performance is a major issue, should copy the input collection, as otherwise external entities could manipulate directly the internal state of this class, eg:
`this.uniqueColumnNames = new ArrayList(uniqueColumnNames)`
furthermore, do we need the ordering? I mean, could use a `Set` which would imply uniqueness

Reply:
Thanks. I looked for all the references to LinkedList and replace them with ArrayLists.